docs/docs/how-to-guides/squared-person-image-support.md

```diff
@@ -0,0 +1,17 @@
+---
+myst:
+  html_meta:
+    "description": "Squared person images in teasers and listings setting"
+    "property=og:description": "Squared person images in teasers and listings setting"
+    "property=og:title": "Squared person images in teasers and listings setting"
+    "keywords": "kitconcept Intranet Distribution, squared person images, TTW"
+---
+
+# Squared person images in teasers and listings setting
+
+The kitconcept intranet distribution provides a setting to enable squared person images in teasers and listings.
```

Comment:
A picture is worth 1000 words... It's great if we can include a screenshot to help show what a feature is.